Gemfile

```diff
@@ -7,11 +7,3 @@ gem "jquery-rails"
 
 # Airbrake must be required before initialization because its middleware is added via an initializer
 gem 'airbrake'
-gem 'rollbar'
-
-group :development, :test do
-  gem 'pry'
-  gem 'pry-nav'
-  gem 'pry-rescue'
-  gem 'pry-stack_explorer'
```

Comment:
still in the gemspec
https://github.com/lumoslabs/smoke_detector/blob/master/smoke_detector.gemspec#L29-L32